simd/src/main/java/com/dylibso/chicory/simd/SimdInterpreterMachine.java

```diff
@@ -234,15 +234,15 @@ private static void F32x4_MUL(MStack stack) {
     }
 
     private static void F32x4_ABS(MStack stack) {
-        var valHigh = stack.pop();
-        var valLow = stack.pop();
+        var offset = stack.size() - 2;
 
-        // https://github.com/tetratelabs/wazero/blob/58488880a334e8bda5be0d715a24d8ddeb34c725/internal/engine/interpreter/interpreter.go#L3168-L3169
-        long resultLow = valLow & (1L << 31 | 1L << 63) ^ 0xFFFFFFFFFFFFFFFFL;
-        long resultHigh = valHigh & (1L << 31 | 1L << 63) ^ 0xFFFFFFFFFFFFFFFFL;
+        var v =
+                LongVector.fromArray(LongVector.SPECIES_128, stack.array(), offset)
+                        .reinterpretAsFloats();
 
-        stack.push(resultLow);
-        stack.push(resultHigh);
+        var result = v.abs().reinterpretAsLongs().toArray();
+
+        System.arraycopy(result, 0, stack.array(), 0, 2);
```

Comment:
Sadly, this sneaked in with other opcodes too, do you mind fixing them all?

This line should be:

```java
System.arraycopy(result, 0, stack.array(), offset, 2);
```

The tests are passing, probably, because we don't have many elements on the stack.
I'd be in favor of refactoring into a method `overrideLastV128`(not a great name though ...) or similar.